templates/cisco_nxos_show_interface_transceiver.textfsm

```diff
@@ -0,0 +1,14 @@
+Value INTERFACE (\S+/\S+)
+Value MANUFACTURER (\S+)
+Value TYPE (.+)
+Value SERIAL (\S+)
+Value PART_NUMBER (\S+)
+
+Start
+  ^${INTERFACE}
+  ^\s+type\s+is\s+${TYPE}
+  ^\s+name\s+is\s+${MANUFACTURER} 
+  ^\s+part\s+number\s+is\s+${PART_NUMBER}
+  ^\s+serial\s+number\s+is\s+${SERIAL} -> Record
+
+EOF
```

Comment:
You can remove the EOF and then if you don't mind adding the following to the end of the **Start** state:
```
  ^\s*$$
  ^. -> Error
```
Once you make the change, you'll most likely have to account for the lines that aren't currently being captured. You can typically just add the first word or two of a line to account for it.

Reply:
@FragmentedPacket thanks for review this PR
I made the changes as suggested